src/style.css

```diff
@@ -0,0 +1,133 @@
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+:root {
+  ---blue: #00acdc;
+}
+
+html {
+  font-family: "Roboto";
```

Comment:
```suggestion
  font-family: Roboto, sans-serif;
```
![Screenshot from 2023-04-03 12-43-24](https://user-images.githubusercontent.com/33029864/229473472-133fbf5d-eef9-4691-89ce-a008a90892cf.png)